scripts/load_notification_templates.py

```diff
@@ -37,12 +37,12 @@ def load_email_templates():
             "plain": None,
         },
         PaymentNotificationType.NEW_BERTH_ORDER_APPROVED: {
-            "html": "berth_offer_new_{lang}.html",
-            "plain": None,
+            "html": "berth_offer_new_{lang}.html",  # Extending base_berth_offer_invoice base-html-template.
```

Comment:
I guess the base template does not yet exist?

Reply:
wops.. Actually, the base tempalte did not work, so the comment is now outdated. Instead of base template, the code is now duplicated. Extending a base or including a snippet did not work in this case when we are using a sandboxed jinja environment and when the templates are handled like they now are. It would need a huge refactor.